lib/tasks/load_tags_from_content_api.rake

```diff
@@ -0,0 +1,39 @@
+desc "Load tags from Content API into Collections Publisher."
+task :load_tags_from_content_api => :environment do
+  require 'gds_api/content_api'
+
+  content_api_endpoint = Plek.new.find("content_api")
+  content_api = GdsApi::ContentApi.new(content_api_endpoint)
+
+  puts "There are currently #{Tag.count} tags in the database"
+  puts "Fetching all tags from the Content API..."
+  tags = content_api.get_list!("#{content_api_endpoint}/tags.json")
+
+  puts "Received a total of #{tags.count} tags"
+  puts "Loading tags into Collections Publisher..."
+  tags.each do |api_tag|
+    unless Tag.where(slug: api_tag.slug).any?
+      tag = Tag.new(slug: api_tag.slug,
```

Comment:
There are currently no tags in collections-publisher, so there's no issue of them not matching the first time this is run. I did this to prevent future runs preventing any changes our editors might have might.